apps/api-example/schema-v1-admin.graphql

```diff
@@ -1926,6 +1927,7 @@ input MemberPlanInput {
   extendable: Boolean!
   maxCount: Int
   availablePaymentMethods: [AvailablePaymentMethodInput!]!
+  migrateToTargetPMid: ID
```

Comment:
Should it not be ID= migrateToTargetPMID as it is for all others?

Reply:
changed it.